resources/Microsoft.NET.SDK.ToolInstaller/Microsoft.NET.SDK.ToolInstaller.psm1

```diff
@@ -0,0 +1,297 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+using namespace System.Collections.Generic
+
+$ErrorActionPreference = "Stop"
+$PSNativeCommandUseErrorActionPreference = $true
+Set-StrictMode -Version Latest
+
+#region Functions
+function Get-DotNetPath {
+    if ($IsWindows) {
+        $dotNetPath = "$env:ProgramFiles\dotnet\dotnet.exe"
```

Comment:
Isn't this for VSCode? If so, I don't believe that supports x86 so you probably don't need that. I assume you also wont' need dotnet\x64.